src/Gui/DlgUnitsCalculator.ui

```diff
@@ -134,6 +143,9 @@
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Copy">
+       <property name="toolTip">
+        <string>copies result into the clipboard</string>
```

Comment:
`Copies`

Reply:
I favor an imperative sentence.
`Copy result into the clipboard`